setup.py

```diff
@@ -83,7 +83,7 @@
             "isort>=5.0,<5.11",
             "jedi>=0.18,<0.19",
             "lxml>=4.6,<4.10",
-            "tox>=3.20,<4.0.0",
+            "tox>=3.20,<4.1",
```

Comment:
Unsure why the dependabot didn't update this version at the same time as the one in the test environment.